csharp/Platform.Collections/ICollectionExtensions.cs

```diff
@@ -16,17 +16,37 @@ public static class ICollectionExtensions
         /// <para>Checking collection for empty.</para>
         /// <para>Проверяет коллекцию на пустоту.</para>
         /// </summary>
+        /// <typeparam name="T">
+        /// <para>The elements collection data type.</para>
+        /// <para>Тип данных элементов коллекции.</para>
+        /// </typeparam>
         /// <param name="collection">
-        /// <para>Method takes an elements collection of <see cref="ICollection<T>"/> type.</para>
-        /// <para>Метода принимает колекцию элементов <see cref="ICollection<T>"/> типа.</para>
+        /// <para>The collection to check.</para>
+        /// <para>Коллекция для проверки.</para>
         /// </param>
         /// <returns>
-        /// <para>Returns a <see cref="bool"/> type variable equal to False if the collection is empty else returns true.</para>
-        /// <para>Возвращает переменную типа <see cref="bool"/> равной false если коллекция пустая иначе возвращает true.</para>
+        /// <para>The <see cref="bool"/> type variable equal to false if the collection is empty else returns true.</para>
+        /// <para>Переменная типа <see cref="bool"/> равна false, если коллекция пуста, иначе возвращает true.</para>
         /// </returns>
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         public static bool IsNullOrEmpty<T>(this ICollection<T> collection) => collection == null || collection.Count == 0;
-
+        
+        /// <summary>
+        /// <para>Checks if all elements of the collection.<see cref="default"/>.</para>
```

Comment:
`Checks if all elements of the collection.<see cref="default"/>.` ->`Checks if all collection elements values are equal to <see cref="default"/> value.`